runtime/tests/error_backtrace1.c

```diff
@@ -3,32 +3,61 @@
 
 HASH_DEFINE_KEY;
 
+// We have to check the function and filename seperately
+// because filepath is not a constant
+static const ExpectedTrace expectedTrace1[] = {
+    { "    func1 ", "error_backtrace1.c:52" },
+    { "    main ", "error_backtrace1.c:65" }
+};
+
+static const ExpectedTrace expectedTrace2[] = {
+    { "    func2 ", "error_backtrace1.c:39" },
+    { "    func1 ", "error_backtrace1.c:61" },
+    { "    main ", "error_backtrace1.c:65" }
+};
+
+static const ExpectedTrace expectedTrace3[] = {
+    { "    func3 ", "error_backtrace1.c:27" },
+    { "    func2 ", "error_backtrace1.c:48" },
+    { "    func1 ", "error_backtrace1.c:61" },
+    { "    main ", "error_backtrace1.c:65" }
+};
+
 void func3() {
-    TaggedPtr taggedErr = _bal_error_construct(makeString("Func3 error"), 7);
+    TaggedPtr taggedErr = _bal_error_construct(makeString("Func3 error"), 27);
     ErrorPtr ep = (ErrorPtr)taggedToPtr(taggedErr);
-    _bal_error_backtrace_print(ep);
-    PC *pcs = (PC *)ep->pcs;
-    uint32_t nPCs = ep->nPCs;
-    assert(nPCs >= 4);
+
+    FILE *fp = fopen("/tmp/backtrace.txt", "w+");
```

Comment:
That's not the right way to create a temp file.

Reply:
Fixed